examples/with-redis/app/form.tsx

```diff
@@ -73,8 +76,8 @@ type FeatureState = {
 };
 
 export default function FeatureForm({ features }: { features: Feature[] }) {
-  const formRef = useRef<HTMLFormElement>(null);
-  const [state, mutate] = useOptimistic(
+  let formRef = useRef<HTMLFormElement>(null);
+  let [state, mutate] = useOptimistic(
```

Comment:
Honest question here, Why change these from const to let? Is that the standard thoughout the repo?

Reply:
It doesn't really matter, use whatever you prefer 😄